packages/cascara/src/components/Table/TableBase.js

```diff
@@ -129,7 +164,24 @@ const TableBase = ({
     ? inferUniqueID(Object.keys(data[0]))
```

Comment:
(Question) Wondering if you need to do this with sortedData instead

Reply:
Thanks for catching this, unique ID inference works with the `data` shape, which isn't impacted by sorting. That's why we didn't change it. But I agree we could use `sortedData` for consistency, changing 👍